packages/@icon-magic/config-reader/src/config-loader.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+const importFresh = require('import-fresh');
+
+/**
+ * Loads a JavaScript configuration from a file.
+ * @param filePath The filename to load.
+ * @returns  The configuration object from the file.
+ */
+function loadJSConfigFile(filePath: string) {
```

Comment:
Does this function have a return interface you can cast the import as? This will help with type safety down the line instead of keeping configs  as `any`.